javascript-web/README.md

```diff
@@ -0,0 +1,40 @@
+# Usage
+### package.json
+```
+dependencies: {
+  "@gomomento/generated-types-webtext": "$current_version",
```

Comment:
Are you sure you are happy with this module name for npmjs publishing?  I'm not challenging it, just want to ensure we are happy with it since it's a one-way-door.

Reply:
it's consistent with the other one; I think, such as it is, it's right.